modules/tensor_mechanics/include/materials/lagrangian/ComputeLagrangianStressPK1.h

```diff
@@ -44,6 +44,4 @@ class ComputeLagrangianStressPK1 : public ComputeLagrangianStressBase
   const MaterialProperty<RankTwoTensor> & _inv_df;
   /// Deformation gradient
   const MaterialProperty<RankTwoTensor> & _F;
-  /// Volume change
-  const MaterialProperty<Real> & _detJ;
```

Comment:
I removed it because I thought it's a little bit ambiguous. Sometimes we need the determinant of the unstabilized defgrad, sometimes of the stabilized defgrad. I guess if the hyperelastic material need det(F), it can just do `_F[_qp].det()`, which has no ambiguity. What do you think?

Reply:
Ya, I changed my mind below.  It's fine this way.